photos.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang='en'>
+  <head>
+    <meta charset='UTF-8'>
+    <meta name='description'>
+    <title>Sedona Photos</title>
+    <link rel="stylesheet" href="">
+  </head>
+
+  <body>
+    <div class="wrapper"><!-- main page -->
+      <header class="page-header">
+        <nav class="main-nav">
+          <ul class="main-nav__list">
+            <li class="main-nav__item"><a href="index.html">Home</a></li>
+            <li class="main-nav__item main-nav__item--active"><a href="photos.html">Photos and videos</a></li>
+            <li class="main-nav__item"><a href="form.html">Feedback Form</a></li>
+            <li class="main-nav__item">HTML Academy</li>
+          </ul>
+          <button class="main-nav__toggle main-nav__toggle--close"></button>
+          <a class="main-nav__link" href="index.html"><img src="" class="main-nav__logo" alt="Sedona logo"></a>
+        </nav>
+
+        <div calss="intro">
+          <ul class="intro-list">
+            <li class="intro-list__item intro-list__item--top">Welcome to the gorgeous</li>
+            <li class="intro-list__item intro-list__item--middle">Sedona</li>
+            <li class="intro-list__item intro-list__item--bottom">"Because the Grand Canyon sucks!"</li>
+          </ul>
+        </div>
+      </header>
+
+      <main class="main-content">
+        <section class="preamble">
+          <div class="preamble__inner">
+            <h2 class="preamble__header">Photo and video</h2>
+            <p class="preamble__paragraph">Can't decide to travel because of the right time? We hope photos will help you forget about politics and economics.</p>
+          </div>
+        </section>
+
+        <div class="gallery">
+          <section class="gallery__main-photo">
+            <a class="gallery_main-img"></a>
+            <div class="gallery__inner">
+              <h3 class="gallery__title">Outback</h3>
+              <p class="gallery__author">Photo by: Boris</p>
+              <p class="gallery__likes">1350</p>
+            </div>
+          </section>
+
+          <section class="gallery__collection">
+            <section class="gallery__photo-vegetation">
+              <a class="gallery_vegetation-img"></a>
```

Comment:
And here I think it's better to have the following `gallery__image gallery__image--vegetation`, so you have some common classes and use modifier just to change the link.